helpers/caching/memory.py

```diff
@@ -0,0 +1,12 @@
+def reclaim_memory():
+    import gc
```

Comment:
Will using imports like this have a performance penalty? 
I guess even if it does, the garbage collection will take orders of magnitude more time.

Reply:
we import torch in threads everywhere, i'm pretty sure it's fine :D